R/gh_fill.R

```diff
@@ -0,0 +1,14 @@
+gh_fill <- function(geohashes, precision) {
+  if (length(unique(nchar(geohashes))) > 1) {
+    stop("Input Geohashes must all have the same precision level.")
+  }
+  if (any(grepl("['ailoAILO]", geohashes))) {
```

Comment:
this allows non-ASCII characters on input, for example.

I would do `if (!all(grepl(<base32>, geohashes)))` instead, and use `ignore.case=TRUE`.

PS why `'`?